client/src/AppBundle/Resources/views/Report/MoneyInShort/category.html.twig

```diff
@@ -7,8 +7,8 @@
 {% set transOptions = {'%client%': report.client.firstname | e } %}
 
 {% block htmlTitle %}{{ ('categoryPage.moneyIn.htmlTitle') | trans }}{% endblock %}
-{% block pageTitle %}{{ ('categoryPage.moneyIn.pageTitle') | trans }}{% endblock %}
-
+{% block pageTitle %}{{ 'form.categoriesIn.label' | trans(transOptions) }}{% endblock %}
```

Comment:
Just a quick check that your intention is to keep this `pageTitle` block, as it most others it has been removed? There are a couple others below too

Reply:
Yeah, each page should have either:
- A `pageTitle` block or
- An empty `pageHeader` block (to hide the default) and a label/legend with `isPageHeading: true`

This ensures that there is exactly one h1 on the page. The second option is for pages which are just a single question (e.g. "Do you have any fees to report?"), with the question itself being the page heading (where we previously might have just had the word "Fees")